streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -176,7 +177,7 @@ class StreamingContextSuite extends FunSuite with BeforeAndAfter with Timeouts w
 
   test("stop gracefully") {
     val conf = new SparkConf().setMaster(master).setAppName(appName)
-    conf.set("spark.cleaner.ttl", "3600")
+    conf.set("spark.cleaner.ttl", "3600ms")
```

Comment:
They were always seconds before, so this test meant "3600 seconds" before. 1 hour = 3600 seconds sounds like the intent rather than 3.6 seconds = 3600ms. I imagine the point of this is "a long TTL"?